common/src/main/resources/ts/utils/graph.ts

```diff
@@ -225,6 +267,45 @@ export class GraphUtils {
         await GraphUtils.renderChartForPDF(newOptions, charts);
     }
 
+    /**
+     * Generate and render graph of the results of a matrix question (for PDF)
+     * @param question  Question object which we want to display the results
+     * @param responses Array of responses which we want to display the results
+     * @param charts    ApexCharts to store and render at the end
+     */
+    static generateCursorChartForPDF = async (question: Question, charts: any,  responses: Responses) : Promise<void> => {
+        if (question.question_type != Types.CURSOR) {
+            return null;
+        }
+
+        let reponses: Response[] = responses.all;
+
+        let resp: number[] = [];
+        let cursorAverage: number;
+
+        // build array with all response
+        for (let r of reponses) {
+            resp.sort(function(a: number, b: number) {
+                return a - b
+            })
+            resp.push(Number(r.answer));
+        }
+        cursorAverage = resp.reduce((a: number, b: number) => a + b, 0) / resp.length;
+
+        // map to build object with response and number of each one
+        const map: Map<number, number> = resp.reduce((acc: Map<number, number>, e: number) =>
+            acc.set(e, (acc.get(e) || 0) + 1), new Map());
+
+        let labels: number[] = Array.from(map.keys());
+        let colors: string[] = ColorUtils.generateColorList(labels.length);
+
+        let newOptions: any = GraphUtils.generateOptions(question.question_type, colors, labels, null, null,
+            null, cursorAverage);
+        newOptions.series = [{ name: "Modalité de réponse", data: Array.from(map.values()) }];
```

Comment:
Tu pourrais utiliser lang.translate('form.modality.response')
```suggestion
        newOptions.series = [{ name: lang.translate('form.modality.response'), data: Array.from(map.values()) }];
```